backstory/src/demo/webapp/gpt2.js

```diff
@@ -11,9 +10,78 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+/* eslint-disable no-unused-vars */
+
```

Comment:
I think all of our JS files are supposed to have headers, which say which page they're for, and which features they implement according to the [style guide](https://docs.google.com/document/d/1IIRpZ9fLD8HodyzIo8H1nRNU0DpwVCp7qFi7upJOsQU/edit#heading=h.1kwviii5lqoe).

Reply:
Done

Reply:
@UdayKalra It'd be great if you could also add the header for this page (as mentioned in style guide), but otherwise looks great.